unofficial.gni

```diff
@@ -193,7 +193,7 @@ template("node_gn_build") {
     }
     if (node_enable_inspector) {
       deps += [
-        "src/inspector:crdtp",
+        "$node_inspector_protocol_path:crdtp",
```

Comment:
The `crdtp` target in `inspector_protocol` can not be built with Node.js (it has dependency on Chromium's things).

The ideal fix would be removing the dependency on Chromium in the upstream `inspector_protocol`, pulling it in Node, and then updating here.